src/stdio_mgr/stdio_mgr.py

```diff
@@ -380,3 +427,4 @@ def __exit__(self, exc_type, exc_value, traceback):
 
 
 stdio_mgr = StdioManager
+_INITIAL_SYS_STREAMS = StdioTuple([sys.stdin, sys.stdout, sys.stderr])
```

Comment:
These should be `sys.__stdin__`, `sys.__stdout__` & `sys.__stderr__`, which should be untouched and we can bypass/undo any wrapping done to `sys.std*`

Reply:
Ah, I hadn't thought about using those, but yeah -- definitely helpful.

It's not a carte-blanche thing, though. We can reference back to those in some cases -- but in general, there may be situations where we might want to restore to the prior `sys.stdfoo`, instead of the canonical `sys.__stdfoo__`, to avoid munging a surrounding stdio mock/wrap.

I think testing `sys.stdfoo is sys.__stdfoo__` should be an unambiguous sign that we're working inside someone else's mocking/wrapping environment, though.  Oh, well, not completely -- if something rummages about in the innards of `sys.stdfoo` instead of replacing the object references, then `is` comparison might still give `True`.

That actually may be an argument against trying to directly reach within `sys.stdfoo` to work with the internal buffer... if `.stdfoo is .__stdfoo__`, then we'd also be tinkering with the "canonical backup".

Reply:
> ... if .stdfoo is .__stdfoo__, then we'd also be tinkering with the "canonical backup".

Yes, this is the dangerous territory that https://github.com/bskinn/stdio-mgr/pull/78 is going in -- IMO the 'dont touch' is cause for concern, lots of it, especially in multi-threading as the tinkering being done isnt atomic, and locking is going to be hard.  https://github.com/bskinn/stdio-mgr/pull/78 may need to be opt-in for a while.